ports/geckolib/glue.rs

```diff
@@ -2216,14 +2214,14 @@ fn fill_in_missing_keyframe_values(all_properties:  &[TransitionProperty],
         return;
     }
 
-    let keyframe = match offset {
-        0. => unsafe {
+    let keyframe = if is_initial {
```

Comment:
@emilio File a follow-up to use an enum. Having a float where only two values matter is even worse, and should have never been r+'d in the first place.